frontend/lib/DataModels/User/user.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:adpv_frontend/DataModels/User/auth_token_response.dart';
+import 'package:adpv_frontend/DataModels/User/user_role.dart';
+import 'package:jwt_decode/jwt_decode.dart';
+
+class User {
+  late AuthTokenResponse tokenResponse;
+  late String email;
+  List<UserRole> userRole = [];
```

Comment:
Powinna być liczba mnoga, bo mamy kilka ról, a nie jedną
```suggestion
  List<UserRole> userRoles = [];
```